xml_converter/generators/cpp_templates/attribute_template.hpp

```diff
@@ -15,19 +15,19 @@
     class XMLError;
 
     enum {{class_name}} {
-        {% for attribute_variable in attribute_variables: %}
-        {{attribute_variable.attribute_name}},
+        {% for attribute_variable in attribute_components %}
```

Comment:
If we are changing this to `attribute_components`, should `attribute_variable` become `attribute_component` as well?

Reply:
Fixed